packages/core/addon/chart-builders/metric.ts

```diff
@@ -24,6 +24,9 @@ import { BaseChartBuilder, C3Row, EmptyC3Data } from './base';
 import { tracked } from '@glimmer/tracking';
 import { MetricSeries } from 'navi-core/models/chart-visualization';
 import NaviFactResponse, { ResponseRow } from 'navi-data/models/navi-fact-response';
+import ColumnFragment from 'navi-core/models/bard-request-v2/fragments/column';
+
+const BLANK_X_VALUE = '';
```

Comment:
Could probably put this in the base one and use it for the raw value of `EmptyC3Data`